components/article/ArticleCardList.vue

```diff
@@ -32,103 +33,71 @@ export default {
   components: {
     ArticleCard
   },
-  props: {},
-  computed: {
-    cardsArticles () {
-      return _.slice(this.articles, 0, 5)
-    },
-    ...mapGetters({
-      articles: `${storePath}/articles`,
-      loading: `${storePath}/loading`,
-      status: `${storePath}/status`,
-      limit: `${storePath}/limit`,
-      error: `${storePath}/error`
-    })
+  props: {
+    articles: {
+      required: true,
+      type: Array
+    }
   },
+  computed: {},
   methods: {},
   mixins: {}
 }
 </script>
 
 <style lang="scss">
-@import "assets/scss/_imports.scss";
+@import "assets/scss/_imports";
 
 .article-card-list {
   display: flex;
   flex-wrap: wrap;
 
-  .article-card {
-    margin: 0 $default-gutters-width $default-gutters-width 0;
+  @media (max-width: $screen-sm) {
+    padding: 0 10px;
```

Comment:
why did you change a variable to px again? It's important because I decided some time ago to change space between components from 20px to 15px. I simply changed gutters variable and said 'Let's magic happen' but nothing happens.. :(